### ISSUE TITLE
Tag check client-side

### ISSUE THREAD

**Issue by el1t** [elmzteam/bydesign#24]
Instead of silently omitting tags with illegal chars and tags that are too long, there should be an indication that illegal tags are present. One solution would be to prevent submission if any single tag is bad.
Also, text should be forcibly transformed to lowercase client-side (and server-side?)

Maybe set a minimum length of 3, since words shorter than 3 letters are likely not legitimate or ambiguous acronyms

**Comment by bluepichu** [comment]
I disagree with tag length - not only is that user error if it's short or ambiguous, but tags like `js` or `c` should be legal.
